Makefile

```diff
@@ -54,12 +56,10 @@ run:
 	docker-compose -p edgex -f docker-compose${NO_SECURITY}${UI}${ARM64}.yml up -d ${SERVICES}
 
 down:
-	docker-compose -p edgex -f docker-compose.yml -f docker-compose-no-secty-with-ui.yml down
+	$(COMPOSE_DOWN)
 
-clean: down
-	-docker rm $$(docker ps --filter "network=edgex_edgex-network" --filter "network=edgex_default" -aq) 2> /dev/null
-	docker volume prune -f && \
-	docker network prune -f
+clean:
+	$(call COMPOSE_DOWN,-v)
```

Comment:
why we want to show the version?

Reply:
It is not version. It is the -v option to `down` which removes the volumes.

Reply:
arh, got you, i missed the `down` directive